### DISCUSSION TITLE
Office Suite

### DISCUSSION THREAD

**Discussion by meximan** [tutao/tutanota#3510] [Ideas]
Is a web browser and android office suite on the road map (something like modified collabora office or only office or even better cryptpad)? If so, when does development begin? Thanks for the awesome service!

**Comment by ghost** [comment]
I think that you are Right, After Finishing Email, Calendar next on the list Encrypted Drive Similar to ProtonDrive after that Encrypted Notes. If we do create an Office Suite like you said, it is more Private GSuite and Totally a Private, End to End Encrypted Office Suite. But needs more time to finish things before as I said

**Comment by meximan** [comment]
@Loki-L1130 thank you!

**Comment by charlag** [comment]
Excuse me what happened here

**Comment by ghost** [comment]
> 
> 
> Excuse me what happened here

@meximan  Requested Tutanota to make or Integrate Office suite like OnlyOffice or CryptPad. Which will make a Completes GSuite focused on Privacy with E2E. I said that it might take Some time for the Team, as they need to finish Mail, Calendar, Drive, Notes (Encrypted) after then they might consider it. I think he thought it won't happen because of the works Tutanota Team already have I said. So, He Closed the Issue.

**Comment by meximan** [comment]
I figured they'll get to it eventually and no reason to continue this thread lol
